.github/workflows/valgrind.yml

```diff
@@ -3,6 +3,7 @@ on:
   pull_request:
     branches:
       - develop
+      - develop2
   schedule:
     # 03:00 every Monday morning
     - cron: '0 3 * * 1'
```

Comment:
Similarly here. This means I think that we _do_ need a separate workflow for this.

Reply:
It looks like that is possible, I will try to do that.